pkg/runner/runner.go

```diff
@@ -26,6 +33,56 @@ func (*CLIRunner) Run(c string, args ...string) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+// Run will use exec.Command as a backing medium and run CombinedOutput.
+func (*CLIRunner) RunWithOutputStream(c string, args ...string) ([]byte, error) {
```

Comment:
This looks familiar. Was it moved from somewhere?

Reply:
yes